index/builder_test.go

```diff
@@ -361,17 +438,17 @@ func TestOptions_FindAllShards(t *testing.T) {
 			compoundShards: [][]zoekt.Repository{
 				{
 					{Name: "repoA", ID: 1},
-					{Name: "sameName", ID: 2},
-					{Name: "sameName", ID: 3},
+					{Name: "repoB", ID: 2},
+					{Name: "repoC", ID: 3},
 				},
 				{
-					{Name: "repoB", ID: 4},
-					{Name: "sameName", ID: 5},
-					{Name: "sameName", ID: 6},
+					{Name: "repoD", ID: 4},
+					{Name: "repoE", ID: 5},
+					{Name: "repoF", ID: 6},
 				},
 			},
 			expectedShardCount: 1,
-			expectedRepository: zoekt.Repository{Name: "sameName", ID: 5},
+			expectedRepository: zoekt.Repository{Name: "something-else", ID: 5},
```

Comment:
fly-by change to make the intent of the test more obvious. Before, the name and id both matched so it was not obvious that we matched by id only.